src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,89 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
+import classNames from 'classnames';
+import { MovieCard } from '../MovieCard';
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+import { MovieData } from '../../types/MovieData';
+import { ResponseError } from '../../types/ReponseError';
+
+type Props = {
+  movies: Movie[],
+  onMoviesChange: (movies: Movie[]) => void,
+};
+
+const emptyImg = 'https://via.placeholder.com/360x270.png?text=no%20preview';
+
+export const FindMovie: React.FC<Props> = ({ movies, onMoviesChange }) => {
+  const [searchQuery, setSearchQuery] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+  const [hasLoadingError, setHasLoadingError] = useState(false);
+  const [movie, setMovie] = useState<Movie>();
+
+  function isAMovieData(obj: MovieData | ResponseError | undefined) {
+    if (!obj) {
+      return false;
+    }
+
+    return 'Poster' in obj && 'Title' in obj
+      && 'Plot' in obj && 'imdbID' in obj;
+  }
+
+  function movieDataToMovie(movieData: MovieData): Movie {
+    const movieFromServer: Movie = {
+      title: movieData.Title,
+      description: movieData.Plot,
+      imgUrl: movieData.Poster === 'N/A'
+        ? emptyImg
+        : movieData.Poster,
+      imdbUrl: `https://www.imdb.com/title/${movieData.imdbID}`,
+      imdbId: movieData.imdbID,
+    };
+
+    return movieFromServer;
```

Comment:
```suggestion
    return {
      title: movieData.Title,
      description: movieData.Plot,
      imgUrl: movieData.Poster === 'N/A'
        ? emptyImg
        : movieData.Poster,
      imdbUrl: `https://www.imdb.com/title/${movieData.imdbID}`,
      imdbId: movieData.imdbID,
    };
```
no need to create a new variable to just return it